exercises/practice/acronym/test/acronym-tests.lfe

```diff
@@ -0,0 +1,26 @@
+(defmodule acronym-tests
+  (behaviour ltest-unit)
+  (export all))
+
+(include-lib "ltest/include/ltest-macros.lfe")
+
+(deftest no-allergies-at-all (is-equal () (allergies:allergies 0)))
```

Comment:
I think you've accidentally left copied and pasted line 7 from the allergies test.